index.html

```diff
@@ -397,6 +397,88 @@ <h2>Manifest</h2>
 
         <section id="versions">
             <h2>Versions</h2>
+            <p>
+                An OCFL Object Inventory MUST include a block for storing versions. This block MUST have the
+                key of <code>versions</code> within the inventory, and it MUST be an array.
+            </p>
+            <p>
+                Within the <code>versions</code> array is a list of JSON objects of type <code>Version</code>.
```

Comment:
I think we want to make it plain that there must be at least one version, I suggest: 

> The <code>versions</code> array MUST be a list of one or more JSON objects of type <code>Version</code>.